db/rdbms/src/main/java/io/camunda/db/rdbms/write/service/RdbmsPurger.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.db.rdbms.write.service;
+
+import io.camunda.db.rdbms.sql.PurgeMapper;
+import java.util.List;
+
+public class RdbmsPurger {
+
+  private static final List<String> TABLE_NAMES =
```

Comment:
@stoerti - A little late, sorry for that, but just a thought: this means whenever a new table is added/removed in the schema, we have to remember to update this list, correct? Seems easy to forget and break something unsuspectingly.

Is there anything we can do to improve this? For example:
- Pulling the table names from some centralized schema file (?), or
- Writing a test which breaks if there is a missing and/or extra table